src/aqua/decider.aqua

```diff
@@ -313,33 +348,32 @@ func poll_new_deals(spell_id: string, listener_id: string, info: AuroraInfo, fro
 
     result <- FluenceAuroraConnector.poll_deals(info.net, info.address, real_from_block)
     if result.success == false:
-        log_err(spell_id, result.error!)    
+        log(["can't receive info about new deals", result.error!])
     else:
         for deal <- result.result:
-            log(spell_id, ["found deal", deal.info.deal_id])
+            log(["found deal", deal.info.deal_id, "from block", deal.block_number])
             if is_worker_created(spell_id, deal.info.deal_id):
-                log(spell_id, ["worker for deal", deal.info.deal_id, "already created"])
+                log(["worker for deal", deal.info.deal_id, "already created"])
             else:
                 process_deal(spell_id, deal)
-            new_from_block <- Json.stringify(deal.block_number)
+            new_from_block <- Json.stringify(deal.next_block_number)
             Spell.set_string("from_block", new_from_block) 
         -- If we found no deals, we check if need to move the from_block forward
         -- We need to do it in case if no deals were done in range of 10000 blocks
         if result.result.length > 1 == false:
             need_update <- need_update_from_block(listener_id, info.net, result.to_block)
             if need_update:
-                log(spell_id, ["updating outdated from_block: [previous from_block, new_from_block]", from_block, result.to_block])
+                log(["updating outdated from_block: [previous from_block, new_from_block]", from_block, result.to_block])
                 to_block_str <- Json.stringify(result.to_block)
                 Spell.set_string("from_block", to_block_str)
 
     
 func main(spell_id: string, listener_id: string, info: AuroraInfo, from_block: string):
     -- Find new deals and create workers
-    --try:
-    poll_new_deals(spell_id, listener_id, info, from_block)
+    try:
+        poll_new_deals(spell_id, listener_id, info, from_block)
 
-    -- TODO: 
-    -- 1. I wonder what will happen when the decider join _many_ deals?
-    --    Will it be able to check them on time?
+    -- TODO: I wonder what will happen when the decider join _many_ deals?
+    --       Will it be able to check them on time?
```

Comment:
The storage gotta work in a way that if execution is stopped _any time_, it will just continue next time _from the same place_